src/__test__/Header.test.js

```diff
@@ -0,0 +1,15 @@
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Header from '../components/Header';
+
+describe('test if Header matches snapshot', () => {
+  it('Check if Header renders correctly', () => {
+    const tree = render(
+      <Router>
+        <Header />
+      </Router>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] As mentioned by the previous reviewer and as required for the project, kindly make sure your App has unit tests for **several components** of the app(i.e.: **not only for the `Header` component**) and some pure functions in Redux code.

- [ ] Also you would make sure that your `App` has integration tests using React Testing Library as required for the project and mentioned by the previous reviewer. I suggest you perform a mocking data fetching and Redux Store connection.